include/robots/norbot.h

```diff
@@ -8,6 +8,14 @@
 // Third-party includes
 #include "third_party/units.h"
 
+#if TEGRA_CHIP_ID == 33
+    #define USER_I2C_DEVICE "/dev/i2c-1"
+#elif TEGRA_CHIP_ID == 24
+    #define USER_I2C_DEVICE "/dev/i2c-0"
+#else
+    #error Unsupported tegra chip id
+#endif
```

Comment:
I think this is a really nice solution. My only reservation is that we now can't test build it on a non-Tegra platform. How about wrapping this like so:
```c++
#ifdef TEGRA_CHIP_ID
... // your code
#else
    #define USER_I2C_DEVICE "/dev/null"
#endif
```

Reply:
...on second thoughts, you could do this without making it Tegra-specific at all. You could just make the default ctor try to open files from ``/dev/i2c-0`` to ``/dev/i2c-$some_number`` and stop when it finds the one that works.

I think @ajc158 might be using the I2C code on a Raspberry Pi and this change would break that. Have I got that right, Alex?